### ISSUE TITLE
 'x' (identifier_name) too short error

### ISSUE THREAD

**Issue by dastan28** [realm/SwiftLint#2134]
Hi guys! 
Recently I've updated my Xcode and OS to High Sierra, but got weird SwiftLint error.
I have code `guard let x = variable else { return }` which gives me an error **Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'x' (identifier_name)** . As far as I know, SwiftLint should skips common variables like **i,x,y** and etc. I've already checked out other closed issues regarding **identifier_name**, but didn't quite get whether the issue is already fixed or not. May you please help?
Details: Xcode 9.3,  High Sierra,  SwifLint: (0.25.0) via Cocoapods

**Comment by marcelofabri** [comment]
SwiftLint started validating local variables with Xcode 9.3. To exclude a variable you can either disable the rule it locally or add exceptions to the configuration:

```yml
identifier_name:
  min_length: # only min_length
    error: 4 # only error
  excluded: # excluded via string array
    - id
    - URL
    - GlobalAPIKey
```

**Comment by RishabhTayal** [comment]
@marcelofabri, I am also getting error in Xcode 9.3. I am getting it for `weak_delegate` rule.

**Comment by dastan28** [comment]
@RishabhTayal what is your code example?

**Comment by marcelofabri** [comment]
@RishabhTayal please open a new issue filling the template

**Comment by RishabhTayal** [comment]
@marcelofabri, I wasn't on the latest version of `swiftlint`. I updated to the latest version and now I am not getting error. Everything is ok. Sorry for false alarm.
